### PR TITLE
Created ad token v4 that uses Base64URLEncoding 

### PR DESCRIPTION
**Pull request by sunnywu**
Changes included:

- Added advertising token v4 implementation
- stop using utcnow/utctimestamp calls as they are not time zone safe (see https://blog.ganssle.io/articles/2019/11/utcnow.html ) and i encounter similar problems running codes from no-utc timezone
- restructure method names and file names to be consistent with other java/c#/c++ uid2 client sdks (an example would be like [this ](https://github.com/IABTechLab/uid2-client-java/pull/16) MR)

### CODE REVIEW

File: tests/test_encryption.py [hunk 1]
```diff
@@ -1,100 +1,237 @@
...
 from uid2_client.keys import *
 
-
 _master_secret = bytes([139, 37, 241, 173, 18, 92, 36, 232, 165, 168, 23, 18, 38, 195, 123, 92, 160, 136, 185, 40, 91, 173, 165, 221, 168, 16, 169, 164, 38, 139, 8, 155])
 _site_secret =   bytes([32, 251, 7, 194, 132, 154, 250, 86, 202, 116, 104, 29, 131, 192, 139, 215, 48, 164, 11, 65, 226, 110, 167, 14, 108, 51, 254, 125, 65, 24, 23, 133])
 _master_key_id = 164
 _site_key_id = 165
 _test_site_key_id = 166
-_site_id = 2001
+_site_id = 9000
```
Comment (sunnywu) [inline tests/test_encryption.py:16]: to be consistent with java/c# sdk

File: tests/test_encryption.py [hunk 2]
```diff
@@ -1,100 +1,237 @@
...
 _site_id2 = 2
 
-_uid2 = 'ywsvDNINiZOVSsfkHpLpSJzXzhr6Jx9Z/4Q0+lsEUvM='
-_now = dt.datetime.utcnow()
+_example_id = 'ywsvDNINiZOVSsfkHpLpSJzXzhr6Jx9Z/4Q0+lsEUvM='
```
Comment (sunnywu) [inline tests/test_encryption.py:19]: to be consistent with java/c# sdk

File: tests/uid2_token_generator.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
+import base64
```
Comment (sunnywu) [inline tests/uid2_token_generator.py:1]: this file will eventually be moved under uid2_client when @jon8787 gets to implement uid2 sharing in this py client.

Comment (jon8787) [reply tests/uid2_token_generator.py:1]: think @zaiweidu and @cody-constine-ttd are working on Sharing in python

**Review by cody-constine-ttd — APPROVED** [review]

File: tests/uid2_token_generator.py (hunk 3 above)
Comment (sunnywu) [reply tests/uid2_token_generator.py:1]: ok

**Review by jon8787 — APPROVED** [review]

**Pull request merged**